README.md

```diff
@@ -1,91 +1,205 @@
+#### 11. Clone this repo & Initiate Terraform
+ and run:
 
-8. Once helm is finished, run `kubectl get services -w` and wait for `gateway` to receive an external IP. It'll look something like this on kubectl or docker for desktop. **Note that minikube doesn't support this**, you'll just have to run `minikube ip` and keep refreshing it with the node port below until it comes up.
+```bash
+terraform init
+```
+
+After a bit of waiting you should get this message:
 
 ```
-gateway                           LoadBalancer   10.102.57.74     123.456.789.123     80:31519/TCP        1m
+Terraform has been successfully initialized!
+
+You may now begin working with Terraform. Try running "terraform plan" to see
+any changes that are required for your infrastructure. All Terraform commands
+should now work.
+
+If you ever set or change modules or backend configuration for Terraform,
+rerun this command to reinitialize your working directory. If you forget, other
+commands will detect it and remind you to do so if necessary.
 ```
 
-At this point you should be able to go to `http://<external ip>` in your browser and see the Magda UI. Note that the search won't work until it's finished indexing regions - to see the progress of this, run `kubectl logs -f -lservice=indexer`. Unless you've got a lot of processing power this will take quite a while - sorry! We're working on making it better.
+#### 12. Edit terraform config
+
+Edit [terraform/magda/terraform.tfvars](./terraform/magda/terraform.tfvars) and supply the follow parameters:
+- `Project id`: the id of the google cloud project that you created (`echo $PROJECT_ID`)
+- `Deploy Region`: which region you want to deploy magda to
+- `credential_file_path`: the path of the service account key file (`key.json`) that we just generated
+- `namespace`: which kubernetes namespace you want to deploy Magda to (generally this should just be "default") 
+- `external_domain`: Optional: what domain you want the Magda server to be accessed from (which requires a bit of extra configuration). Leave blank to just access your instance through a temporary domain. You can set this later if necessary.
+
+Other optional settings and their default values (if not set) are:
+- `cluster_node_pool_machine_type`: The machine type to use, see https://cloud.google.com/compute/vm-instance-pricing for more details. Default: `n1-standard-4`
+- `kubernetes_dashboard`: Whether turn on [kubernetes_dashboard](https://github.com/kubernetes/dashboard) or not; Default: `false`
 
-By default, data.gov.au will be crawled on startup so you'll start with some data.
+You can find full list of configurable options from [here](./terraform/magda/variables.tf).
+
+#### 13. Edit default helm config
+
+Look at [config.yaml](./config.yaml). It has reasonable defaults but you might want to edit something - it will give you a new instance with a standard colour scheme/logos and no datasets (yet).
```

Comment:
Again, I think it makes more sense to make it a text and not a link.
```suggestion
Look at `config.yaml`. It has reasonable defaults but you might want to edit something - it will give you a new instance with a standard colour scheme/logos and no datasets (yet).
```